docs/projects/ad719x_asdz/index.rst

```diff
@@ -16,14 +16,18 @@ Supported boards
 -------------------------------------------------------------------------------
 
 - :adi:`EVAL-AD7190 <EVAL-AD7190>`
+- :adi:`EVAL-AD7195 <EVAL-AD7192>`
 - :adi:`EVAL-AD7193 <EVAL-AD7193>`
+- :adi:`EVAL-AD7195 <EVAL-AD7194>`
```

Comment:
I think it's a typo, as it would be 3 times AD7195.
Also, could you change lines 18-22 to be in this way?
:adi:\`EVAL-AD719...\`
It will render the same, just that it's simpler written in code.

Reply:
Done.